openforcefield/utils/toolkits.py

```diff
@@ -1789,78 +1784,34 @@ def from_rdkit(self, rdmol, allow_undefined_stereo=False):
         from rdkit import Chem
         from openforcefield.topology.molecule import Molecule
 
-        # Check for undefined stereochemistry
-        from rdkit.Chem import EnumerateStereoisomers
-        # TODO: Does this work for molecules with 3D geometry?
-        unspec_stereo = False
-        # Use a copy of the input, in case EnumerateStereochemstry changes anything in-place
-        rdmol_copy = Chem.Mol(rdmol)
```

Comment:
I think we still want to work with a copy of the molecule in this function -- Many of the things we do may change the rdmol passed in by the user.

Reply:
Good catch! I completely missed that copy.